src/styles/blocks/catalog.scss

```diff
@@ -18,6 +18,27 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    @keyframes scale {
+      from {
+        transform: scale(1, 1);
+      }
+      to {
+        transform: scale(1.2, 1.2);
+      }
+    }
+
+    @keyframes color {
+      100% {
+        color: #34568b;
+      }
+    }
+
+    transform: color, scale;
```

Comment:
Did you mean transition?
https://developer.mozilla.org/en-US/docs/Web/CSS/transition